app/webpacker/components/calendar.js

```diff
@@ -87,25 +87,26 @@ document.addEventListener('turbolinks:load', function() {
           $el.addClass("fc-event-small");
         };
         $el.addClass("fc-event-"+ info.event.extendedProps.status);
-        $el.attr("data-rightbar", "true");
+        if (info.event.extendedProps.unclickable != true){
```

Comment:
you could just write `if (!info.event.extendedProps.unclickable)` no?

Or you could also go the other way around and short circuit like we do in ruby: `if (info.event.extendedProps.unclickable) return` 

the advantages are : 
- less codediff in the PR
- less indentation 

the downside is maybe some lisibility loss